packages/flutter_test/lib/src/window.dart

```diff
@@ -601,6 +577,39 @@ class FakeAccessibilityFeatures implements ui.AccessibilityFeatures {
   }
 }
 
+/// Used to fake insets and padding for [TestFlutterView]s.
+///
+/// See also:
+///
+///   * [TestFlutterView.padding], [TestFlutterView.systemGestureInsets],
+///     [TestFlutterView.viewInsets], and [TestFlutterView.viewPadding] for test
+///     properties that make use of [FakeViewPadding].
+///   * DEPRECATED: [TestWindow.viewInsetsTestValue], [TestWindow.viewPaddingTestValue],
+///     [TestWindow.paddingTestValue], [TestWindow.systemGestureInsetsTestValue],
+///     for test value setters that can make use of [FakeViewPadding].
+class FakeViewPadding implements ui.ViewPadding {
```

Comment:
This actually came _from_ the tests. I found it in the `window_test` file if I remember correctly. I grabbed it from there and moved it here.

I kept it as "fake" because of the other data class in this file, `FakeAccessibilityFeatures`. I had assumed it was "Fake" for data and "Test" for classes that actually do things.

Reply:
I forgot about FakeAccessibilityFeatures. Looks like we already have a naming pattern for these then. Calling this FakeViewPadding is perfect then!